### ISSUE TITLE
Second duplicated echo is ignored

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#672]
### Issue summary

It seems that if you echo the same string twice in a row, only one text is printed on the terminal.

### Expected behavior
That everything is printed.

### Actual behavior
Only one string is printed.

### Steps to reproduce

```javascript
term.echo('hello').echo('hello')
```

**Comment by jcubic** [comment]
The problem was with cache, this also printed two strings:
```javascript
term.echo('foo').echo('bar').echo('foo')
```

**Comment by trankten** [comment]
Having the same issue. 
Reverting to version 2.25.0 fixes it till a new version is up.
https://unpkg.com/jquery.terminal@2.25.0/js/jquery.terminal.min.js

Thank you.

**Comment by jcubic** [comment]
Released in 2.27.0
